dependencies.gradle

```diff
@@ -99,7 +99,7 @@ ext.libs = [
                 'flipperNetworkPlugin'    : "com.facebook.flipper:flipper-network-plugin:$flipper",
         ],
         element     : [
-                'opusencoder'             : "io.element.android:opusencoder:1.0.4",
+                'opusencoder'             : "io.element.android:opusencoder:1.1.0",
```

Comment:
any notable changes with the dependency update?

Reply:
`OggOpusEncoder` is now an interface to make mocking it easier. Also, we had to separate the old `external fun setBitrate` into `fun setBitrate` and `external fun setEncoderBitrate` as `mockk` can't mock external funs.